### PR TITLE
Document the allowNull property on the boolean transform

### PR DESCRIPTION
**Pull request by bmac**

### CODE REVIEW

**Comment by locks** [comment]
When was this option introduced?

**Comment by bmac** [comment]
http://emberjs.com/blog/2016/06/09/ember-data-2-6-released.html
Ember Data 2.6.

File: addon/-private/transforms/boolean.js [hunk 1]
```diff
@@ -22,6 +22,20 @@ const { isNone } = Ember;
   });
   ```
 
+  By default the boolean transform only allows for values of `true` or
+  `false`. You can opt into allowing `null`/`undefined` values for
```
Comment (pangratz) [inline addon/-private/transforms/boolean.js]: Technically `null` is returned when the serialized value `isNone`.

**Comment by pangratz** [comment]
Thank you kindly

**Pull request merged**